lib/nerves_hub_link/certificate.ex

```diff
@@ -64,7 +64,22 @@ defmodule NervesHubLink.Certificate do
           "[NervesHubLink] Using default system certificates. Requests may fail if the NervesHub server certificate is not signed by a globally trusted CA, or the installed system certificates are old."
         )
 
-        :public_key.cacerts_get()
+        maybe_public_certs()
+    end
+  end
+
+  # TODO: We can drop this check once OTP 25
+  # is our lowest supported version
+  if String.to_integer(System.otp_release()) < 25 do
```

Comment:
```suggestion
  if String.to_integer(System.otp_release()) >= 25 do
```
The comparison looks flipped. Once this is fixed, you should also be able to turn the `apply` to a direct call.

Reply:
Oh no 😭 Thank you, fixed!